### ISSUE TITLE
Add a stop button in the Pipeline Overview view

### ISSUE THREAD

**Issue by kevin-messer-jemmic** [jenkinsci/pipeline-graph-view-plugin#478]
### What feature do you want to see added?

In the pipeline overview and in the pipeline console, there is a button Rebuild and a button Console.

I would like to have also a button Stop which will cancel the current run I am exploring.

This would help to fill even more the gaps with Blue Ocean.

**Comment by felipecrs** [comment]
I agree, Blue Ocean's stop button is very handy and would be great to have it here too.

**Comment by felipecrs** [comment]
I guess this one can be beside the green "Rebuild" button.

**Comment by felipecrs** [comment]
Hm... I think we can afford a new button. Rebuild while the build is still running works and can be useful in some situations.

For example, I have an 10h long pipeline with a bunch of optional parallel stages.

I can rebuild once I see some failing but still letting the build finish to receive earlier feedback on remaining stages.
